src/stackedtime/sim_nr.jl

```diff
@@ -187,33 +206,37 @@ function damping_br81(; delta::Real=0.1, rateK::Real=10.0, lambda_min::Real=1e-5
         ::Union{Nothing,Factorization,AbstractMatrix{Float64}}=nothing,
         ::Union{Nothing,AbstractVector{Float64}}=nothing
     )
-        # @printf "    it=%d, λ=%g, nF=%g\n" it λ nF
-        it < 1 && (bigK = 0.0; return true, 1.0)
+        # Initialization step
+        it < 1 && (bigK = 0.0; return true, λ_max)
+        
         if nF2_it != it
-            # first time we're called this iteration 
+            # First time we're called in this iteration
             nF2 = norm(F, 2)  # store the residual 
             nF2_it = it
             return false, calc_λ()
```

Comment:
calc_lambda should respect/enforce lambda_max, or is that option only in the signature to make all the signatures align?